### PR TITLE
January Ratification Poll outcomes

### PR DESCRIPTION
**Pull request by blimpa**

### CODE REVIEW

File: MIP55/MIP55c3-Subproposals/MIP55c3-SP14.md [hunk 1]
```diff
@@ -80,8 +80,9 @@ All claims and payouts will be managed according to a process contained in the C
 ```
 Official Contact : @layerzero
 Contact Email/Handle: layer.zero@ses.makerdao.network
-Date Added: (yyyy-mm-dd)
+Date Added: 2023-01-23
 Total Amount: In Dai. 5M DAI
 Wallet Address: Public wallet address of Multi-Sig.  TBD
```
Comment (Gala-G) [inline MIP55/MIP55c3-Subproposals/MIP55c3-SP14.md:85]: when is this supposed to be determined?

**Review by Gala-G — COMMENTED** [review]
I need to see your other PR. Did you add the new keeper network to mip63?

**Comment by Gala-G** [comment]
LGTM
the only thing I can think of right now is adding Chainlink Automation  to MIP63 - will do

File: MIP38/mip38.md [hunk 2]
```diff
@@ -540,6 +514,16 @@ Offboarded Core Units are archived at the end of the list under *Offboarded Core
...
+#### Governance Communications (COM-001)
+
+- **Summary** - The Governance Communications Core Unit performs a set of programs that focus on MakerDAO's informational accessibility, stakeholder coordination, and communication practices as a decentralized and public organization.
+- **Ratification Date for the Initial Proposal Set** - 2021-07-26
+- **Ratification Date for Offboarding** - 2023-01-23
+- - **Related Proposals** [see](https://mips.makerdao.com/mips/list?search=$%23cu-com-001&orderDirection=ASC&hideParents=true&)
+
+---
+
+
```
Comment (ghost) [inline MIP38/mip38.md]: unnecessary empty line

```suggestion
```

File: MIP4/MIP4c2-Subproposals/MIP4c2-SP31.md [hunk 3]
```diff
@@ -8,11 +8,11 @@ MIP to be amended: MIP63
 Author(s): Sam MacPherson (@hexonaut)
 Contributors:
 Tags: MIP63, facilitator
```
Comment (ghost) [inline MIP4/MIP4c2-Subproposals/MIP4c2-SP31.md]: why is facilitator a tag here?

File: MIP55/MIP55c3-Subproposals/MIP55c3-SP14.md (hunk 1 above)
Comment (ghost) [reply MIP55/MIP55c3-Subproposals/MIP55c3-SP14.md:85]: The wallet address? Just needs done before the exec. @prose11 can chase up with layerzero this week

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]
LGTM feel free to merge when ready

**Pull request merged**